### PR TITLE
OSDOCS-2467: Creates a list of the SRE-supported services.

### PR DESCRIPTION
**Pull request by EricPonvelle**
This PR addresses [OSDOCS-2467](https://issues.redhat.com/browse/OSDOCS-2467). This PR adds a new assembly with modules to the Policy and Service Definitions book.

JIRA: https://issues.redhat.com/browse/OSDOCS-2467

Preview: https://deploy-preview-36468--osdocs.netlify.app/openshift-dedicated/latest/support/osd-managed-resources.html

### CODE REVIEW

File: osd_policy/osd-supported-services.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+:context: osd-supported-services
+
+toc::[]
+
+[id="osd-supported-services-overview"]
+= Overview
+
+The following YAML file covers all of the services supported by Service Reliability Engineers (SRE).
+
+== List of supported services
```
Comment (dustman9000) [inline osd_policy/osd-supported-services.adoc:13]:
```suggestion
== Full list of managed resources
```

File: osd_policy/osd-supported-services.adoc [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+----
+
+== List of supported addon services
+
+[source,yaml]
+----
+include::https://raw.githubusercontent.com/openshift/managed-cluster-config/master/resources/addons-namespaces/main.yaml[]
+----
+
+== List of supported webhooks
```
Comment (dustman9000) [inline osd_policy/osd-supported-services.adoc:27]:
```suggestion
== Objects protected by validating webhooks
```

**Review by dustman9000 — COMMENTED** [review]
Some suggestions for the addons and webhooks headers

File: _topic_map.yml [hunk 3]
```diff
@@ -310,6 +310,8 @@ Topics:
   File: policy-understand-availability
 - Name: Update life cycle
   File: osd-life-cycle
+- Name: OpenShift Dedicated supported Services
```
Comment (dustman9000) [inline _topic_map.yml:313]:
```suggestion
- Name: OpenShift Dedicated Managed Resources
```

File: osd_policy/osd-supported-services.adoc [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+= {product-title} supported services
+include::modules/attributes-openshift-dedicated.adoc[]
+:context: osd-supported-services
+
+toc::[]
+
+[id="osd-supported-services-overview"]
+= Overview
+
+The following YAML file covers all of the services supported by Service Reliability Engineers (SRE).
```
Comment (dustman9000) [inline osd_policy/osd-supported-services.adoc:11]:
```suggestion
The following contents file covers all resources managed by Service Reliability Engineers (SRE). Customers should not attempt to modify these resources.
```

File: osd_policy/osd-supported-services.adoc [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+= {product-title} supported services
+include::modules/attributes-openshift-dedicated.adoc[]
+:context: osd-managed-resources
+
+toc::[]
+
+[id="osd-managed-resources-overview"]
+= Overview
+
+The following contents file covers all resources managed by Service Reliability Engineers (SRE). Customers should not attempt to modify these resources.
```
Comment (dustman9000) [inline osd_policy/osd-supported-services.adoc:11]:
```suggestion
The following content covers all resources managed by the Service Reliability Engineering Platform (SRE-P) Team. Customers should not attempt to modify these resources.
```

File: osd_policy/osd-managed-resources.adoc [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+= {product-title} supported services
+include::modules/attributes-openshift-dedicated.adoc[]
+:context: osd-managed-resources
+
+toc::[]
+
+[id="osd-managed-resources-overview"]
+= Overview
+
+The following contents file covers all resources managed by Service Reliability Engineers (SRE). Customers should not attempt to modify these resources.
```
Comment (dustman9000) [inline osd_policy/osd-managed-resources.adoc:11]:
```suggestion
The following covers all resources managed or protected by the Service Reliability Engineering Platform (SRE-P) Team. 
```

File: osd_policy/osd-managed-resources.adoc [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+:context: osd-managed-resources
+
+toc::[]
+
+[id="osd-managed-resources-overview"]
+= Overview
+
+The following covers all resources managed or protected by the Service Reliability Engineering Platform (SRE-P) Team.  Customers should not attempt to modify these resources as doing so will potentially lead to cluster instability.
+
+== Full list of managed resources
```
Comment (dustman9000) [inline osd_policy/osd-managed-resources.adoc:13]:
```suggestion
== OSD Managed Resources
```

File: osd_policy/osd-managed-resources.adoc [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+[%collapsible]
+====
+[source,yaml]
+----
+include::https://raw.githubusercontent.com/openshift/managed-cluster-config/master/resources/managed/all-osd-resources.yaml[]
+----
+====
+
+== List of namespaces managed by OSD addons
```
Comment (dustman9000) [inline osd_policy/osd-managed-resources.adoc:23]:
```suggestion
== OSD Addons Namespaces
```

File: osd_policy/osd-managed-resources.adoc [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+
+[%collapsible]
+====
+[source,yaml]
+----
+include::https://raw.githubusercontent.com/openshift/managed-cluster-config/master/resources/addons-namespaces/main.yaml[]
+----
+====
+
+== Objects protected by validating webhooks
```
Comment (dustman9000) [inline osd_policy/osd-managed-resources.adoc:33]:
```suggestion
== OSD Validating Webhooks
```

File: osd_policy/osd-managed-resources.adoc [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+
+[%collapsible]
+====
+[source,json]
+----
+include::https://raw.githubusercontent.com/openshift/managed-cluster-validating-webhooks/master/docs/webhooks-short.json[]
```
Comment (dustman9000) [inline osd_policy/osd-managed-resources.adoc:39]: Let's use the full json instead:
```suggestion
include::https://raw.githubusercontent.com/openshift/managed-cluster-validating-webhooks/master/docs/webhooks.json[]
```

File: osd_policy/osd-managed-resources.adoc [hunk 11]
```diff
@@ -0,0 +1,41 @@
+[id="osd-managed-resources"]
+= {product-title} supported services
```
Comment (dustman9000) [inline osd_policy/osd-managed-resources.adoc:2]:
```suggestion
= {product-title} managed resources
```

**Review by dustman9000 — COMMENTED** [review]
Some minor formatting and content changes.

**Review by bergerhoffer — COMMENTED** [review]
Just a few things!

**Pull request merged**